tests/quic.c

```diff
@@ -1174,6 +1174,88 @@ static int test_quic_server_hello(int verbose) {
     return ret;
 }
 
+#if defined(HAVE_ALPN) && defined(HAVE_SNI)
```

Comment:
When building with only `--enable-quic`, the new test is missing some of APIs available only when `--enable-opensslall` is configured:
```
/usr/bin/ld: tests/unit_test-quic.o: in function `test_quic_alpn':
.../wolfssl//tests/quic.c:1221: undefined reference to `wolfSSL_CTX_set_servername_callback'
/usr/bin/ld: .../wolfssl//tests/quic.c:1222: undefined reference to `wolfSSL_CTX_set_servername_arg'
collect2: error: ld returned 1 exit status
```
I'd recommend including `defined(OPENSSL_ALL)` in the conditions protecting this new test

Reply:
Ah, missed the outer `#ifdef' several pages above. Added by the recent push.

Reply:
Resolved the leak that was caused by an early return. All green now.